code/modules/power/supermatter/supermatter.dm

```diff
@@ -1084,9 +1084,9 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			if(GRAVITATIONAL_ANOMALY)
 				new /obj/effect/anomaly/grav(L, 250)
```

Comment:
Why would this not also affect gravitational anomalies?

Reply:
Those things aren't pretty much dangerous

Reply:
Also you would want the flux and grav one go away faster, because those things are very annoying